tests/e2e/util.go

```diff
@@ -2082,11 +2082,17 @@ func getK8sMasterIP(ctx context.Context, client clientset.Interface) string {
 			addrs := node.Status.Addresses
 			for _, addr := range addrs {
 				if addr.Type == v1.NodeExternalIP {
+					if (net.ParseIP(addr.Address)).To4() == nil {
```

Comment:
you can move this check to line: 2084 ? 
for ex: 
```
addr.Type == v1.NodeExternalIP  && net.ParseIP(addr.Address)).To4() != nil {
```

Reply:
Done